### PR TITLE
add README.md

### LINKED ISSUE #95: Add README for tinysql 3.0

### PR DESCRIPTION
**Pull request by rebelice**
close #95

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+  - Iconic function: the ability to handle simple SELECT statements
+  - Knowledge topic:
+        - parser
+        - data mapping from the relational model to KV
+        - generating operator
+- Stage 2: insert and update
+  - Target: the ability to write data using KV engine API
+  - Iconic function: the ability to handle simple INSERT/UPDATE statements
+  - Knowledge topic:
+    - volcano model
```
Comment (tisonkun) [inline README.md:22]: volcano model already introduced by SELECT stmt?

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
...
+
+### Build
+
+```
+make
+```
+
+### Run & Play
+
+Use `./bin/tinysql-server` to run the server and use MySQL client `mysql --host 127.0.0.1 --port 4000 -u root` to connect the server.
```
Comment (tisonkun) [inline README.md:60]: where is the kv engine? does this command run the sql engine only?

File: README.md (hunk 1 above)
Comment (rebelice) [reply README.md:22]: In order to reduce the difficulty of the first stage, we will not introduce the volcano model by `select stmt`

Comment (tisonkun) [reply README.md:22]: fine.

File: README.md (hunk 2 above)
Comment (rebelice) [reply README.md:60]: Yes, this command only runs the sql engine. However, the sql engine will have unistore as a mock store.

Comment (tisonkun) [reply README.md:60]: Get it.

**Review by tisonkun — APPROVED** [review]
LGTM. Merging...

**Pull request merged**